packages/core/src/lib/input/input.svelte

```diff
@@ -80,7 +79,7 @@ $: icon = {
 
   {#if icon !== ''}
     <span
-      class={cx('absolute right-2 top-1.5', {
+      class={cx('absolute right-2', {
```

Comment:
Fixes the icon issue seen here: 
![image](https://github.com/viamrobotics/prime/assets/1930371/e3bc50ec-7789-4be8-81ed-f3eb5c603c30)

Reply:
Does this have any impact on other icon use?

Reply:
@ethanlook These are the only three icons allowed on the input, they are based on the `state` prop.